<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>500</height>
    </size>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>9</y>
      <width>87</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="red_btn">
         <property name="text">
          <string>Add Employee</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="blue_btn">
         <property name="text">
          <string>List All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="yellow_btn">
         <property name="text">
          <string>Search and Edit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="menu_btn">
       <property name="text">
        <string>Menu</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>89</x>
      <y>9</y>
      <width>401</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>400</width>
         <height>400</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="home">
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>50</y>
           <width>321</width>
           <height>161</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>13</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>wellcome to employee manager. Please select from left side menu</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="red">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(170, 255, 127);</string>
        </property>
        <widget class="QLineEdit" name="fname">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>40</y>
           <width>161</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MV Boli</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;MV Boli&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>First Name</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lname">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>70</y>
           <width>161</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MV Boli</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;MV Boli&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Last Name</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="code">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>100</y>
           <width>161</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MV Boli</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;MV Boli&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>National Code</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MV Boli</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;MV Boli&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Date of birth</string>
         </property>
        </widget>
        <widget class="QPushButton" name="camera_btn">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>170</y>
           <width>41</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>pic/icons8-camera-48.png</normaloff>pic/icons8-camera-48.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>170</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MV Boli</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;MV Boli&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Take a Picture</string>
         </property>
        </widget>
        <widget class="QPushButton" name="submit_btn">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>230</y>
           <width>91</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>13</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(170, 0, 0);</string>
         </property>
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
        <widget class="QLabel" name="lbl_error">
         <property name="geometry">
          <rect>
           <x>210</x>
           <y>240</y>
           <width>181</width>
           <height>16</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 0, 0);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLineEdit" name="birthday">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>130</y>
           <width>161</width>
           <height>21</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="blue">
        <property name="styleSheet">
         <string notr="true">background-color: blue</string>
        </property>
       </widget>
       <widget class="QWidget" name="yellow">
        <property name="styleSheet">
         <string notr="true">background-color: yellow</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
